src/util/util.js

```diff
@@ -19,16 +20,24 @@ exports.find = (buf, byte) => {
   return -1
 }
 
+const timestampWithOffsetToISOString = (timestamp, offset) => strftime.timezone(offset)('%FT%T%:z', new Date(timestamp * 1000))
```

Comment:
optional nit: extract `'%FT%T%:z'` to a constant

Reply:
👍 will do